_includes/meta.html

```diff
@@ -163,8 +166,23 @@
       })
     }
 
-    function render_result(elementId, content, append = true){
-      document.getElementById(elementId).innerHTML = (append == true) ? content : `<div></div>`;
+    function render_result(elementId, content, append = false) {
+        const currentHTML = document.getElementById(elementId).innerHTML
+        document.getElementById(elementId).innerHTML = (append == true) ? currentHTML + content : content;
```

Comment:
Fixed HTML render helper when we aren't appending new content. I didn't see examples where we're clearing the inner HTML of an element to an empty div but if the need arises, I suggest making a new method to clear the display of an HTML and just making the inner html to an empty string rather than adding an empty div